guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -1,45 +1,33 @@
 [id="upgrading_a_connected_{project-context}_server_{context}"]
 = Upgrading a connected {ProjectServer}
 
-Use this procedure for a {ProjectServer} with access to the public internet
+Upgrading a {ProjectServer} with access to the public internet includes backing up the server, ensuring that the system and {Project} configuration is up-to-date, and running the upgrade.
 
 include::snip_warning-maintain-config-noop.adoc[]
 
-include::snip_prerequisites-upgrading-project-server.adoc[]
-
-.Upgrade {ProjectServer}
-
-. Stop all {Project} services:
+.Procedure
+. Create a backup of your {ProjectServer}:
+.. Stop all {Project} services:
 +
 [options="nowrap" subs="+quotes,verbatim,attributes"]
 ----
 # {foreman-maintain} service stop
 ----
-. Take a snapshot or create a backup:
+.. Take a snapshot or create a backup:
 * On a virtual machine, take a snapshot.
 * On a physical machine, create a backup.
 +
-. Start all {Project} services:
+.. Start all {Project} services:
 +
 [options="nowrap" subs="+quotes,verbatim,attributes"]
 ----
 # {foreman-maintain} service start
 ----
```

Comment:
I never liked this step, as it makes it "too complicated" for the physical case.

For the VM case, "stop services", "take snapshot", "start services" is correct.
But for the physical one, the user really only has to do `f-m backup …` and it will take care of services and everything.

I know, that's not a section you wrote/changed, but do you think we could improve it too? :)